TestBenches/TrackletProcessor_test.cpp

```diff
@@ -128,7 +128,7 @@ std::cout<<module_name[MODULE_];
   std::cout << "Loaded the input files:\n";
   for (unsigned i = 0; i < nInnerStubMems; i++)
    //(i+2)%nInnerStubMems has the innerstubs match the order in the emulation
```

Comment:
@jasonfan393 I think this comment is obsolete now. Or maybe it should be moved to TrackletProcessor.h?
```suggestion
```